src/loaders/RasterJsonLoader.js

```diff
@@ -1,7 +1,11 @@
+/* eslint-disable */
```

Comment:
I think this may be leftover from me editing the code and forgetting to remove and run lint-fix